CDTS Blazor/Data/Services/AzureBlobConnectionFactory.cs

```diff
@@ -51,6 +51,7 @@ public async Task<CloudBlobContainer> GetBlobContainer(string container = null)
 
         private CloudBlobClient GetClient()
         {
+            // TODO : fail gracefully when connection string is invalid
             if (!CloudStorageAccount.TryParse(connectionString: this.connectionString, out CloudStorageAccount cloudStorageAccount))
             {
                 throw new CloudStorageAccountConnectionStringException();
```

Comment:
You should consider adding localized exception message. [Reference](DL: Will this be used for usability or just further user research? )).

Reply:
The exception message is in the resources. You can look at the exception class itself it is there 👍